scripts/build.js

```diff
@@ -69,7 +69,7 @@ async function main() {
   // copy webpack buildin
   await copy(
     __dirname + "/../node_modules/webpack/buildin/*.js",
-    __dirname + "/../dist/buildin/"
+    __dirname + "/../dist/ncc/buildin/"
```

Comment:
In the process, the buildins directory got moved... I think this is because __dirname is being renormalized, but I didn't track this cause completely.